Dockerfile.onbuild

```diff
@@ -16,7 +16,8 @@ ONBUILD COPY package.json yarn.lock* .yarnrc* .npmrc* npm-shrinkwrap.json* packa
 # This is in one giant command to keep the image size small
 # TODO: When Finnbuild uses Docker 1.13, we can use --squash, which means this won't have to be one giant command
 ONBUILD RUN apk upgrade -U && \
-    apk add --no-cache --virtual build-dependencies make gcc g++ python git && \
+    apk add --no-cache --virtual build-dependencies make gcc g++ python git || \
+    apk add --no-cache --virtual build-dependencies make gcc g++ python3 git && # 'python' isn't available in alpine3.12+ \
```

Comment:
Why try python first and fall back to python3, instead of just always using python3?

Reply:
Because python2 is what is already being provided now, so giving 3 instead of 2 could break things for everyone.